docs/examples.md

```diff
@@ -0,0 +1,180 @@
+# Examples
+
+<!-- TODO: improve the following -->
```

Comment:
I think the examples should be made more clear and explain what they are showing.

Also some (like the Edge API Routes) are just copy-pasted from the next.js docs, I don't see much value in that (if we want we can just provide a link instead of the actual code?).

Reply:
Originally, it was mostly about showing the features we supported. I'm happy to remove them if our support for them is documented well elsewhere.

Reply:
I see 🙂, as I said for example the Edge API Routes one indeed is, I think it's no super high priority but I think we could improve things here 🙂

It could be also nice to maybe provide examples like of how to use KVs etc...? 🤔